src/cryptonote_core/tx_pool.h

```diff
@@ -308,10 +313,11 @@ namespace cryptonote
      *
      * @param h the hash of the transaction to get
      * @param tx return-by-reference the transaction blob requested
+     * @param tx_relay last relay method us
      *
-     * @return true if the transaction is found, otherwise false
+     * @return Status of transaction
```

Comment:
That makes things less clear.

Reply:
Failed to revert this, now it should be reverted properly.